<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="LogsBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1021</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GetButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>107</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="FromFileLabel">
       <property name="text">
        <string>Файл:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FromFileInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="FromLinkLabel">
       <property name="text">
        <string>Ссылка:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FromLinkInput"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
